diff --git a/src/modules/coral2_dws.py b/src/modules/coral2_dws.py
index 29ca500..2c22b89 100755
--- a/src/modules/coral2_dws.py
+++ b/src/modules/coral2_dws.py
@@ -383,9 +383,7 @@ def _workflow_state_change_cb_inner(workflow, jobid, winfo, handle, k8s_api):
             "job-manager.dws.resource-update",
             payload={
                 "id": jobid,
-                "resources": directivebreakdown.apply_breakdowns(
-                    k8s_api, workflow, resources, _MIN_ALLOCATION_SIZE
-                ),
+                "resources": resources,
             },
         ).then(log_rpc_response)
     elif state_complete(workflow, "Setup"):
@@ -730,13 +728,6 @@ def main():
     # start watching k8s workflow resources and operate on them when updates occur
     # or new RPCs are received
     with Watchers(handle, watch_interval=args.watch_interval) as watchers:
-        init_rabbits(
-            k8s_api,
-            handle,
-            watchers,
-            args.resourcegraph,
-            args.disable_compute_node_draining,
-        )
         services = register_services(handle, k8s_api)
         watchers.add_watch(
             Watch(k8s_api, WORKFLOW_CRD, 0, workflow_state_change_cb, handle, k8s_api)